docs/integrations/integration-docs.md

```diff
@@ -15,6 +15,10 @@ Documentation is a critical step that assists customers who may use your integra
 The documentation must conform to the [Integration Doc Format](doc-structure).
```

Comment:
@fvigo [This](https://deploy-preview-291--demisto-content-docs.netlify.app/docs/integrations/integration-docs) leads to nowhere... any can we remove this?

Reply:
Yes, good catch. We can maybe add an example of HelloWorld instead: https://github.com/demisto/content/blob/master/Packs/HelloWorld/Integrations/HelloWorld/README.md

Reply:
@fvigo I think that there are enough example here https://deploy-preview-291--demisto-content-docs.netlify.app/docs/integrations/integration-docs#documentation-examples

Reply:
ok